Library/SharedFunctions.swift

```diff
@@ -660,3 +660,28 @@ public func isRewardDigital(_ reward: Reward?) -> Bool {
     existingReward.shipping.preference
     .isAny(of: Reward.Shipping.Preference.none)
 }
+
+public func estimatedShippingText(for reward: Reward, selectedShippingRule: ShippingRule) -> String {
+  guard reward.shipping.enabled else { return "" }
+
+  /// Make sure the current reward has shipping rules and that one of them matches the selected shipping rule (from the locations dropdown).
+  guard let shippingRules = reward.shippingRules,
+        let currentRewardShippingRule = shippingRules
+        .first(where: { $0.location.country == selectedShippingRule.location.country })
+  else {
+    return ""
+  }
+
+  guard let estimatedMin = currentRewardShippingRule.estimatedMin?.amount.rounded(.towardZero),
+        let estimatedMax = currentRewardShippingRule.estimatedMax?.amount.rounded(.towardZero),
+        estimatedMin > 0 || estimatedMax > 0 else {
+    return ""
+  }
+
+  /// Drop digits after the decimal.
+  let formattedMin = String(format: "%.0f", estimatedMin)
+  let formattedMax = String(format: "%.0f", estimatedMax)
+
+  // TODO: Update string with translations
+  return "About $\(formattedMin)-$\(formattedMax)"
```

Comment:
I think this needs to check if `formattedMin == formattedMax` and just use ex `About $\(formattedMin)$` if they're equal. Will you fix and maybe add tests for this function specifically, especially if it's not already covered by other tests?